services/src/main/scala/cromwell/pipeline/service/ProjectConfigurationService.scala

```diff
@@ -31,21 +25,21 @@ object ProjectConfigurationService {
     new ProjectConfigurationService {
 
       def addConfiguration(projectConfiguration: ProjectConfiguration, userId: UserId): Future[Unit] = {
-        def toTuple(config: ProjectFileConfiguration): (Path, Map[String, TypedValue]) =
-          config.path -> config.inputs.map(fileParameter => fileParameter.name -> fileParameter.typedValue).toMap
+        def toTuple(wdlParams: WdlParams): (Path, Map[String, TypedValue]) =
+          wdlParams.path -> wdlParams.inputs.map(fileParameter => fileParameter.name -> fileParameter.typedValue).toMap
 
-        def toProjectFileConfiguration(path: Path, inputs: Map[String, TypedValue]): ProjectFileConfiguration =
-          ProjectFileConfiguration(path, inputs.map((FileParameter.apply _).tupled).toList)
+        def toProjectFileConfiguration(path: Path, inputs: Map[String, TypedValue]): WdlParams =
```

Comment:
I think do you mean ?
```suggestion
        def toWdlParams(path: Path, inputs: Map[String, TypedValue]): WdlParams =
```

Reply:
Oops, fixed